### PR TITLE
Allow  Wrapped Token withdraw

### PR DESCRIPTION
**Pull request by nkrulikovsky**
## What does this PR do?

enable to withdraw wrapped token from meta pool.

NOTE
Withdrawing function is having more issue like the following.
- cannot withdraw one token
- max value don't display over the TokenInput ( we are giving Zero to underlying token on meta pool )

### CODE REVIEW

File: src/hooks/useApproveAndWithdraw.ts [hunk 1]
```diff
@@ -33,6 +36,19 @@ export function useApproveAndWithdraw(
   const { gasStandard, gasFast, gasInstant } = useSelector(
     (state: AppState) => state.application,
   )
+
+  const POOL = POOLS_MAP[poolName]
```
Comment (ug02fast) [inline src/hooks/useApproveAndWithdraw.ts:40]: Can you use basicPoolContext pools here?

File: src/hooks/useWithdrawFormState.ts [hunk 2]
```diff
@@ -37,17 +37,16 @@ export type WithdrawFormAction = {
   fieldName: FormFields | "reset"
   address?: string
   value: string
+  max?: BigNumber //TODO: get underlying token balance from registry pool
```
Comment (ug02fast) [inline src/hooks/useWithdrawFormState.ts:40]: Can you use registry pool and get the underlying token balance?

**Review by ug02fast — CHANGES_REQUESTED** [review]
Nice work. Can you leverage the registry pools now instead of POOLS_MAP?

File: src/hooks/useApproveAndWithdraw.ts (hunk 1 above)
Comment (nkrulikovsky) [reply src/hooks/useApproveAndWithdraw.ts:40]: Good callout!

File: src/hooks/useWithdrawFormState.ts (hunk 2 above)
Comment (nkrulikovsky) [reply src/hooks/useWithdrawFormState.ts:40]: On `BasicPoolsProvider.tsx`, we are getting underlyingTokenBalances on line 249.

Comment (ug02fast) [reply src/hooks/useWithdrawFormState.ts:40]: great, is there a problem with using pools from registry then?

Comment (ug02fast) [reply src/hooks/useWithdrawFormState.ts:40]: Should the TODO: comment still be there?

Comment (nkrulikovsky) [reply src/hooks/useWithdrawFormState.ts:40]: Yes, on line 112, we are getting pools from getPoolsBaseData on BasicPoolsProvider.
So, underlyingTokenBalance is Zero now.

**Pull request closed without merging**